application/modules/seo/models/Redirect.php

```diff
@@ -119,22 +120,26 @@ public static function generateRedirectFile()
         $head .= " * This file is generated automatically\n";
         $head .= "*/\n";
         $redirects = [
-            'static' => ArrayHelper::map(
-                self::find()->where(
-                    'type = :type AND active = :active',
-                    [':type' => self::TYPE_STATIC, ':active' => true]
-                )->all(),
+            'static' =>  ArrayHelper::map(
+                $query = (new Query)
+                    ->select('from, to')
+                    ->from(self::tableName())
+                    ->where(['active' => true])
+                    ->andWhere(['type' => self::TYPE_STATIC])
+                    ->all(),
                 'from',
                 'to'
             ),
```

Comment:
May be rewrite it like as:

```php
'static' => static::find()
    ->select(['to', 'from'])
    ->where(['active' => true, 'type' => self::TYPE_STATIC])
    ->asArray(true)
    ->indexBy('from')
    ->column(),
```

1. We don't use ArrayHelper
2. We use standard method for searching